flag/service/resource/vaultcrt/vaultrt.go

```diff
@@ -0,0 +1,6 @@
+package vaultcrt
+
+type VaultCrt struct {
+	ExpirationThreshold string
+	Namespace           string
+}
```

Comment:
This is trying to make it right in the first place, plus boyscouting. ExpirationThreshold is a new option and Namespace already existed but was only hard coded in the resource implementation.